cardano-api/src/Cardano/Api/Orphans.hs

```diff
@@ -247,6 +248,10 @@ instance Crypto.Crypto crypto => ToJSON (Shelley.RewardUpdate crypto) where
                           , "nonMyopic" .= Shelley.nonMyopic rUpdate
                           ]
 
+instance Crypto.Crypto crypto => ToJSON (Shelley.PulsingRewUpdate crypto) where
+  toJSON (Shelley.Pulsing _ _) = "calculating"
```

Comment:
is this alright?

Reply:
I think if we're not interested in the intermediate calculation `Aeson.Null` would probably be more appropriate.

Reply:
There is only a `ToJSON` instance and no `FromJSON` instance. Is this because the output is only intended as debugging output for consumption by 3rd party tools? Or something along those lines?

Reply:
that's my understanding @DavidEichmann , but @Jimbo4350 would know better.

Reply:
> There is only a `ToJSON` instance and no `FromJSON` instance. Is this because the output is only intended as debugging output for consumption by 3rd party tools? Or something along those lines?

yes. almost every SPO is using the JSON output for daily operation, calculations, leaderlogs, etc...   lol